libyul/backends/wasm/WordSizeTransform.cpp

```diff
@@ -169,7 +170,8 @@ void WordSizeTransform::operator()(Block& _block)
 					if (BuiltinFunction const* f = m_inputDialect.builtin(std::get<FunctionCall>(*assignment.value).functionName.name))
 						if (f->name == "datasize"_yulstring || f->name == "dataoffset"_yulstring)
 						{
-							yulAssert(f->literalArguments && f->literalArguments.value()[0], "");
+							yulAssert(f->literalArguments.size() == 1, "");
+							yulAssert(f->literalArguments[0] == LiteralKind::String, "");
```

Comment:
Well couldnt these be merged into a single assert?

Reply:
I just split 'em, so that the lines don't get too long :-D... if they're split, then you know which one fails... but feel free to merge them back together :-). Or should I :-D?